public/source/main.js

```diff
@@ -53,6 +53,7 @@ ko.bindingHandlers.autocomplete = {
       var lastChar = value.slice(-1);
       if (lastChar == '/' || lastChar == '\\') {  // When "/" or "\"
         server.getPromise('/fs/listDirectories', {term: value}).then(function(directoryList) {
+          $(element).val(directoryList.shift() + ungit.config.fileSeparator);
```

Comment:
I think the `ungit.config.fileSeparator` should only be added if it isn't already the last character.
Because if I enter on windows `C:\` it autocompletes to `C:\\`.

Reply:
will do